### PR TITLE
Upload trigger files on successful upload of data files to print house

### PR DESCRIPTION
**Pull request by gjvis**
`CSVUploadJob` is a weird name, its a class that encapsulates the logic of turning a `Batch` into everything needed to send to the print house (i.e. file contents, file paths). Can anyone think of a better name?

### CODE REVIEW

**Comment by gjvis** [comment]
Ok, apart from the weird naming (which might be fine?) I think this is ready for review.

File: app/models/csv_upload_job.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+  def trigger_path
+    upload_path('trg')
+  end
+
+  private
+
+  def upload_path(extension)
+    timestamp = batch.created_at.strftime('%Y%m%d%H%M%S%L')
+    "/data.in/pensionwise_output_#{timestamp}.#{extension}"
```
Comment (andrewgarner) [inline app/models/csv_upload_job.rb]: Is this case sensitive? If so it should be `/Data.in`

Comment (gjvis) [inline app/models/csv_upload_job.rb]: It well may be, nice catch.

File: app/models/csv_upload_job.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  def payload_path
+    upload_path('csv')
+  end
+
+  def trigger
+    ''
+  end
+
+  def trigger_path
+    upload_path('trg')
```
Comment (andrewgarner) [inline app/models/csv_upload_job.rb]: The current UAT environment has `/Trigger.in` - does that need to change?

Comment (gjvis) [inline app/models/csv_upload_job.rb]: That directory exists as part of their "standard" SFTP setup. For this integration they're expecting trigger files alongside the data files.

Comment (andrewgarner) [inline app/models/csv_upload_job.rb]: :+1:

**Comment by andrewgarner** [comment]
:+1:

**Pull request merged**